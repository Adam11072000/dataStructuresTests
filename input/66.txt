init 6981
getHighestScoredLabel 0 6991
quit