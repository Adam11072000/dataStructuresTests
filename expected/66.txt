init done.
deleteImage: FAILURE
addImage: INVALID_INPUT
deleteImage: FAILURE
setLabelScore: FAILURE
mergeSuperPixels: INVALID_INPUT
setLabelScore: FAILURE
deleteImage: INVALID_INPUT
addImage: SUCCESS
addImage: SUCCESS
getHighestScoredLabel: FAILURE
mergeSuperPixels: INVALID_INPUT
setLabelScore: SUCCESS
addImage: SUCCESS
quit done.
